docs/dns-challenge/nsupdate.md

```diff
@@ -0,0 +1,85 @@
+# Variables for nsupdate dns-challenge
+
+None
```

Comment:
I know we don't have it for all providers, but would you add a table describing the provider specific variables here. like in `docs\dns-challenge\azure.md`

Reply:
added